src/Symfony/Bridge/Monolog/Handler/ConsoleHandler.php

```diff
@@ -108,6 +126,9 @@ public function close(): void
      */
     public function onCommand(ConsoleCommandEvent $event): void
     {
+        $input = $event->getInput();
+        $this->setInput($input);
```

Comment:
`$this->input` must be unset in the `close` function.

Also, shorter code:
```suggestion
        $this->setInput($event->getInput());
```